network/betweenness.go

```diff
@@ -87,3 +93,57 @@ func Betweenness(g graph.Graph) map[int]float64 {
 
 	return cb
 }
+
+// BetweennessWeighted returns the betweenness centrality for nodes in the weighted
+// graph g used to construct the given shortest paths.
+//
+//  C_B(v) = \sum_{s ≠ v ≠ t ∈ V} (\sigma_{st}(v) / \sigma_{st})
+//
+func BetweennessWeighted(g graph.CostGraph, p search.ShortestPaths) map[int]float64 {
+	nodes := g.NodeList()
+	cb := make(map[int]float64, len(nodes))
```

Comment:
Is there any reason why in Betweennes() the nodes are explicitly inserted into cb while here they are not?